modules/installation-configuration-parameters.adoc

```diff
@@ -2071,7 +2071,7 @@ Use the `bootMACAddress` configuration setting to enable Ironic to identify the
 |The URL for communicating with the host's BMC controller.
 The address configuration setting specifies the protocol.
 For example, `redfish+http://10.10.10.1:8000/redfish/v1/Systems/1234` enables Redfish.
-For more information, see "BMC addressing" in the "Deploying installer-provisioned clusters on bare metal" section.
+For more information, see "BMC addressing" in the "Installing on bare metal with the IPI Installer" section.
```

Comment:
The original phrasing is more accurate

Reply:
Original phrasing is, however, inconsistent with all other titles that begin with `Installing...`

Reply:
Addressed